x-pack/legacy/plugins/index_lifecycle_management/public/sections/edit_policy/components/hot_phase/hot_phase.js

```diff
@@ -254,6 +278,39 @@ export class HotPhase extends PureComponent {
                             defaultMessage: 'hours',
                           }),
                         },
+                        {
+                          value: 'm',
+                          text: i18n.translate('xpack.indexLifecycleMgmt.hotPhase.minutesLabel', {
+                            defaultMessage: 'minutes',
+                          }),
+                        },
+                        {
+                          value: 's',
+                          text: i18n.translate('xpack.indexLifecycleMgmt.hotPhase.secondsLabel', {
+                            defaultMessage: 'seconds',
+                          }),
+                        },
+                        {
+                          value: 'ms',
+                          text: i18n.translate(
+                            'xpack.indexLifecycleMgmt.hotPhase.millisecondsLabel',
+                            {
+                              defaultMessage: 'milliseconds',
+                            }
+                          ),
+                        },
+                        {
+                          value: 'micros',
+                          text: i18n.translate('xpack.indexLifecycleMgmt.hotPhase.microsLabel', {
+                            defaultMessage: 'micros',
```

Comment:
nit: I think I would spell out `microseconds`

Reply:
@alisonelizabeth 
>It looks like some unrelated changes may have snuck in on this PR. Can you revert the changes >made to template_form_schemas.tsx and validate_policy.ts?

Would doing a `git rebase master` on my local  `bugfix/ILM_add_policy_units` do the trick?